### DISCUSSION TITLE
implementing diffisuvity without Kh

### DISCUSSION THREAD

**Discussion by ghost** [Parcels-code/Parcels#994] [Q&A]
Hi everyone,,
does anyone know how to implement diffusion when using a fieldset (derived from HYCOM) that does that does not have the fields Kh_zonal and Kh_meridional?

Many thanks

**Comment by reint-fischer** [comment]
Hi @molkjam,

Diffusion or eddy diffusivity can be modelled to capture possible motion at a scale smaller than your model (HYCOM) resolution. When not included in the model output, the eddy diffusivity is often estimated from literature for your specific purposes.

 The simplest option is to add a field with a constant eddy diffusivity by using [`fieldset.add_constant_field`](https://oceanparcels.org/gh-pages/html/#parcels.fieldset.FieldSet.add_constant_field) in which you can specify the value with `'kh_zonal'`. 

Another option is to find or create another data source with the spatially varying eddy diffusivity and add that via [`fieldset.add_field`](https://oceanparcels.org/gh-pages/html/#parcels.fieldset.FieldSet.add_field).

Hope this helps!
